.gitignore

```diff
@@ -1,4 +1,4 @@
-/target
+target/
```

Comment:
I agree with @psiroky, add exceptions will be better solution, then copy-paste `/target` from project to project for hundreds of time. As other benefit - if you have `/target` package - you will never forget to add exception, but it is so easy to forget to add gitignore with `target` to every project.